app/uk/gov/hmrc/customsservicestatusfrontend/views/DashboardPage.scala.html

```diff
@@ -45,22 +43,21 @@
             @Html("<b>" + Messages(s"dashboard.available.planned.p2") + "</b>" + " " + Messages(s"dashboard.available.planned.p2.fromDate", s"${Formatters.instantFormatHours(plannedWork.startDateTime)}", s" ${Formatters.instantFormatDate(plannedWork.startDateTime)}"))
         </p>
         <p class="govuk-body">
-            @Html("<b>" + Messages(s"dashboard.available.planned.p3") + "</b>" + " " + Messages(s"dashboard.available.planned.p3.toDate", s"${Formatters.instantFormatHours(plannedWork.endDateTime.get)}", s" ${Formatters.instantFormatDate(plannedWork.endDateTime.get)}"))
+            @Html("<b>" + Messages(s"dashboard.available.planned.p3") + "</b>" + " " + Messages(s"dashboard.available.planned.p3.toDate", s"${Formatters.instantFormatHours(plannedWork.endDateTime.getOrElse(throw new Exception("endDateTime missing")))}", s" ${Formatters.instantFormatDate(plannedWork.endDateTime.getOrElse(throw new Exception("endDateTime missing")))}"))
         </p>
         <p class="govuk-body">
             @plannedWork.commsText match {
-            case DetailType.CommsText(html) => {@Html("<b>" + Messages("dashboard.available.planned.p4") + "</b>" + " " + Messages(s"dashboard.available.planned.p4.details", s"${html}"))
-                }
+                case DetailType.CommsText(html) => {@Html("<b>" + Messages("dashboard.available.planned.p4") + "</b>" + " " + Messages(s"dashboard.available.planned.p4.details", s"$html"))}
             }
         </p>
     }
 }
 
 @layout(pageTitle = Some(s"$pageTitlePrefix - ${messages("service.name")} - ${messages("service.title.suffix")}")) {
 
- @if(state == AVAILABLE) {
+@if(state == AVAILABLE) {
    @availableContent
- } else if(state == UNAVAILABLE) {
+ } else if (state == UNAVAILABLE) {
```

Comment:
minor: another pre-existing issue, but if it's possible to change these to type-safe `===` , that would be great.